models/eprop_synapse_bsshslm_2020.h

```diff
@@ -582,6 +596,13 @@ eprop_synapse_bsshslm_2020< targetidentifierT >::set_status( const DictionaryDat
   {
     // We must pass here if called by SetDefaults. In that case, the user will get and error
     // message because the parameters for the synapse-specific optimizer have not been accessed.
+    auto optimizer_dict = getValue< DictionaryDatum >( d->lookup( names::optimizer ) );
+    auto it = optimizer_dict->find( names::optimize_each_step );
+    if ( it != optimizer_dict->end() )
+    {
+      throw BadProperty(
+        "Optimization in every time step optimize_each_step is not available for eprop_synapse_bsshslm_2020." );
```

Comment:
```suggestion
        "eprop_synapse_bsshslm_2020 only supports optimize_each_step == False." );
```

Reply:
Fixed in [44f0c09](https://github.com/nest/nest-simulator/pull/3207/commits/44f0c09f15cd822a46c4f71890d97827fe2325a2)